components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -48,6 +48,16 @@
   color: $theme.notificationSuccessColor !important;
   border-color: $theme.notificationSuccessColor !important;
 }
+
+input[type="checkbox"] {
+  outline: 0 none !important;
+  box-shadow: none !important;
+  -moz-transform: scale(.85) !important;
```

Comment:
There's a layout bug in Firefox only, arising from the fact that the checkboxes are larger than in other browsers.